backend-models/service-tests/showDuggaservice-test.txt

```diff
@@ -0,0 +1,393 @@
+showDuggaservice.php
+
+—------------------------------------------------------------------------------------------------------------------------
+TEST #1 - 100-105
+Get active users
+
+***
+Pre-req: -
+***
+Send parameters inputs to showDuggaservice.php
+    send
+    {
+        $opt = “UPDATEAU”
+        $hash = hash
+
+    }
+
+    SQL-query: SELECT active_users FROM groupdugga WHERE hash=hash
+    This should show all active users
+
+***
+Gather service output
+***
+    Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
+    This is the expected output for the micro service.
+
+—------------------------------------------------------------------------------------------------------------------
+TEST #2 - 106-110
+Create active users
+
+***
+Pre-req: $active == NULL
```

Comment:
this is a pre-req that is true, but which circumstance guaranties that this value is NULL ?

Reply:
hash supplied is not in the database! (i have double checked this)

Reply:
you have manually checked the database yes, but that is no a guarantee that such a value wont be added in the future does it?

Reply:
If we dont supply a predetermined hash and AUtoken what value should we have?

Reply:
you have a couple of hash values that you could choose from. from there just make a query to the database to check if there is any matches for a specific hash. if there is just move to the next one, or something like that?

Reply:
hash is the important of the two, with it being a primary key and all, therefor the minimum that has to be done is to check to see if that value already exists in the database

Reply:
solved

Reply:
You now check the database which is good. But what happens if the SELECT query is true?
add couple more hashes that can be cycled trough.

Reply:
The purpose of the query is explained in the comment. The same comment can still be applied even if we add any hash. Therefor the tester has to be responsible for chaning the hash. Also as discussed in the discord the tests are assumed to use the init db, which does not include this hash. I do not see further example hashes as a solution.